Unix/tests/pal/test_pal.cpp

```diff
@@ -32,14 +32,14 @@
 #include <pal/hashmap.h>
 #include <pal/format.h>
 
-#if (defined(sun) || defined(hpux) || defined(aix)) && defined(CONFIG_ENABLE_WCHAR)
+#if defined(sun) || defined(hpux) || defined(aix)
 
```

Comment:
Again, if CONFIG_ENABLE_WCHAR is never defined, then this if should always fail.  Note it's &&, not ||

Reply:
you are right. I have deleted this section.

Reply:
You didn't delete?